doc/publishing-api-syntactic-usage.md

```diff
@@ -90,7 +90,7 @@ Requests to update an existing draft content item:
   - `content_id` the primary identifier for the content to publish.
   - `type` the type of unpublishing to create/perform.
 
-### Other request params:
+### Optional request params:
```

Comment:
These aren't all optional though, some of them are required depending on the unpublishing type.